recipes/boltzmannclean/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "e4b4ed0a7ae9c1f3dfb9283c87fd1c846d801934f16afcf0e09ce5f24bc8d073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
Please remove the `fn` line, it is not needed anymore.

Reply:
done in c84720eec24f3112dc0a900ad739b0c2c78d948b